src/Microsoft.VisualStudio.Editors/PropPages/AssemblyInfoPropPage.vb

```diff
@@ -69,7 +69,6 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                     datalist.Add(data)
                     data = New PropertyControlData(VsProjPropId80.VBPROJPROPID_AssemblyCompany, "Company", Me.Company, ControlDataFlags.PersistedInAssemblyInfoFile, New Control() {Me.CompanyLabel})
                     datalist.Add(data)
-                    datalist.Add(data)
```

Comment:
This was causing an assert about multiple items having same DISPID. Seems like a typo - can't think of any reason why this would need to be added twice.